tests/CppUTestExt/IEEE754PluginTest_c.c

```diff
@@ -25,6 +25,10 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <CppUTest/CppUTestConfig.h>
+
+#ifdef CPPUTEST_HAVE_FENV
```

Comment:
Perhaps it doesn't :-). Actually I was hoping you might know some clever way to run into infinity without having to use a #define at all.... like FLOAT_MAX * 2 or something (I'm not really serious, because that would just be a different define ;-). Because it would be nice if this file contained nothing but plain C code.